pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -907,6 +907,11 @@ func (p *PriorityQueue) Update(logger klog.Logger, oldPod, newPod *v1.Pod) error
 	p.lock.Lock()
 	defer p.lock.Unlock()
 
+	if _, ok := p.inFlightPods[newPod.UID]; ok {
+		logger.Error(nil, "The pod is currently being processed, skip update", "pod", klog.KObj(newPod))
```

Comment:
no need to have error logs

Reply:
maybe change it to info log with log level 6?

Reply:
done